src/lib/room.js

```diff
@@ -41,8 +41,10 @@ export const onChatClose = async () => {
 	if (!isMobile()) {
 		store.setState({ minimized: true });
 		parentCall('minimizeWindow');
+		route('/');
+	} else {
+		route('/chat-finished');
 	}
```

Comment:
After this change, I see a blank white page issue in multiple tabs. Same steps as https://github.com/WideChat/Rocket.Chat/issues/1159

Earlier Instead of blank, it was a chat-finished page, but the user had a chance to start a new chat using the `New Chat` button. But now user is stuck.

Reply:
@ear-dev @bhardwajaditya X is not visible when the screen is blank

<img width="822" alt="Screenshot 2022-05-24 at 9 16 32 PM" src="https://user-images.githubusercontent.com/18264684/170077821-2a453bf4-3e26-4934-bee8-ec2179964139.png">

Reply:
UPDATE: Sometimes I see X sometimes I don't. So I think we should wait for the two tabs issue Aditya is working on and merge both together?

Reply:
I agree that it is best to wait for the fix that @bhardwajaditya has recommended.  This issue:  https://github.com/WideChat/Rocket.Chat/issues/1199

@bhardwajaditya can you please address this issue next?  And once it is ready we'll want to review all of the two tab scenarios in order to verify that it's handling our issue, and that it's not making any other issue worse.....  thanks!